http-core/src/main/java/org/apache/pekko/http/javadsl/model/headers/CacheDirectives.java

```diff
@@ -28,7 +28,7 @@ public static CacheDirective MAX_STALE() {
         return new org.apache.pekko.http.scaladsl.model.headers.CacheDirectives.max$minusstale(OptionConverters.toScala(Optional.empty()));
     }
     public static CacheDirective MAX_STALE(long deltaSeconds) {
-        return new org.apache.pekko.http.scaladsl.model.headers.CacheDirectives.max$minusstale(OptionConverters.toScala(OptionalLong.of(deltaSeconds)));
+        return new org.apache.pekko.http.scaladsl.model.headers.CacheDirectives.max$minusstale(new scala.Some(deltaSeconds));
```

Comment:
Since `deltaSeconds` cannot be null (its a Java primitive) we can just directly create a `scala.Some` here.